spec/models/card_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+describe Card do
+  it 'create card with empty original text' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.